backend/apps/core/locks.py

```
@@ -0,0 +1,140 @@
+
+    def _release(self) -> bool:
+        if not self.acquired:
+            return False
+        try:
+            current_value = cache.get(self.lock_key)
+            if current_value == self.lock_value:
+                cache.delete(self.lock_key)
+                self.acquired = False
+                return True
+            return False
+        except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)